### ISSUE TITLE
Add a `clever diag` command that outputs infos for support

### ISSUE THREAD

**Issue by hsablonniere** [CleverCloud/clever-tools#341]
Something that outputs:

* the version of clever-tools
* the OS
* how it was installed (deb, rpm, nupkg/choco, npm i -g, just tar gz)
* node version

**Comment by hsablonniere** [comment]
OK, proposition is: `clever version` or `clever -V` with the verbose option:

* `clever version -v`
* `clever -Vv`

```
[DEBUG] Load configuration from environment variables
[DEBUG] Load configuration from /home/jsmith/.config/clever-cloud
clever-tools  1.5.2
Node.js       v12.5.0
Platform      linux
Architecture  x64
Exec path     /home/jsmith/.nvm/versions/node/v12.5.0/bin/node
Packaged      false
```

```
[DEBUG] Load configuration from environment variables
[DEBUG] Load configuration from /home/jsmith/.config/clever-cloud
clever-tools  1.5.2
Node.js       v12.2.0
Platform      linux
Architecture  x64
Exec path     /path/to/clever
Packaged      true
```

**Comment by hsablonniere** [comment]
Fixed by #342
